fhirflat/ingest.py

```diff
@@ -574,6 +579,35 @@ def convert_data_to_flat(
                 f"Errors saved to {resource.__name__.lower()}_errors.csv"
             )
 
+    if parallel:
+        total_t = timeit.default_timer()
+        _ = Parallel(n_jobs=-1)(
```

Comment:
if/else might not be needed as n_jobs=1 should be equivalent to a for loop according to [joblib.Parallel](https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html)